scripts/test/hyriseBenchmarkTPCC_test.py

```diff
@@ -30,15 +29,14 @@ def main():
   arguments = {}
   arguments["--scale"] = "1"
   arguments["--time"] = "60"
-  arguments["--runs"] = "100"
```

Comment:
No idea about TPC-C. Is that an intended change? Is one run sufficient?

Reply:
`runs` defaults to infinity. 100 was way too low.